### ISSUE TITLE
Saving Tut 3a to JSON - Object of type FractionalVolume is not JSON serializable

### ISSUE THREAD

**Issue by lrebscher** [MetaCell/NetPyNE-UI#391]
Imported Tut 3a, created and simulated an experiment.

netpyne sim throws an exception with message: `Object of type FractionalVolume is not JSON serializable`

**Comment by salvadord** [comment]
@joewgraham - I think you have been working on saving models that include rxd?

**Comment by joewgraham** [comment]
Yeah, and everything should work with Pickle saving...  I want to talk in the dev meeting today about the need for JSON saving of sims/data.  If it's needed, then I think we should use something like `jsonpickle` to serialize stuff, but that will require some implementation...
